examples/interaction/js_callbacks/customjs_for_tools.py

```diff
@@ -21,10 +21,15 @@
            tools="box_select", x_range=(0, 1), y_range=(0, 1))
 
 # using Quad model directly to control (non)selection glyphs more carefully
-quad = Quad(left='left', right='right',top='top', bottom='bottom',
+quad = Quad(left='left', right='right', top='top', bottom='bottom',
             fill_alpha=0.3, fill_color='#009933')
 
-p.add_glyph(source, quad, selection_glyph=quad, nonselection_glyph=quad)
+p.add_glyph(
+    source,
+    quad,
+    selection_glyph=quad.clone(fill_color='blue'),
+    nonselection_glyph=quad.clone(fill_color='gray'),
```

Comment:
With this change and adding the "TapTool" to the tools, this example makes more sense to me. In the newer version a click on a renderer has a visual effect.